src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -1115,8 +1115,11 @@ namespace cryptonote
     
     struct response_t: public rpc_response_base
     {
+      std::string block_id;
+
       BEGIN_KV_SERIALIZE_MAP()
         KV_SERIALIZE_PARENT(rpc_response_base)
+        KV_SERIALIZE(block_id)
```

Comment:
You should increase `CORE_RPC_VERSION_MINOR` if you change RPC definitions.

Reply:
Good point, updated